ReadMe.md

```diff
@@ -1,3 +1,34 @@
-# MAGE SDK for JavaScript (ES5)
+# MAGE SDK (Web)
 
-This is a work-in-progress.
+This is the official [MAGE](https://www.npmjs.com/package/mage) SDK for web browsers.
+
+- Completely written in ES5 to be compatible with the widest range of browsers.
+- Uses CommonJS (`require()` calls and `module.exports`).
+
+## Compatibility
+
+This version of `mage-sdk-js` works with **MAGE >=1.1.0**.
+
+## Installation
+
+In the example below, we assume you use `NPM` to install dependencies for your web application.
```

Comment:
But some people may use Bower, etc. or plain old `<script>` tags We are not compatible. Perhaps I can be more explicit about that?